File: Python/model/jogo.py
class Jogo:

    __id = int
    __nome = str
    __genero = str
    __ano = int
    __tenho = False
    __favorito = False

    def __init__(self, nome, genero, ano):
        self.__nome = nome
        self.__genero = genero
        self.__ano = ano

    def setId(self, id):
        self.__id = id

    def setNome(self, nome):
        self.__nome = nome

    def setGenero(self, genero):
        self.__genero = genero

    def setAno(self, ano):
        self.__ano = ano

    def setTenho(self, tenho):
        self.__tenho = tenho

    def setFavorito(self, favorito):
        self.__favorito = favorito

    def getId(self):
        return self.__id

    def getNome(self):
        return self.__nome

    def getGenero(self):
        return self.__genero

    def getAno(self):
        return self.__ano

    def getTenho(self):
        return self.__tenho

    def getFavorito(self):
        return self.__favorito

    def __str__(self):
        return f'{self.__id}. Nome = {self.__nome}, Gênero = {self.__genero}, Ano = {self.__ano}, Tenho = {"Sim" if self.__tenho else "Não"}, ' \
               f'Favorito = {"Sim" if self.__favorito else "Não"}'

File: PHP/Jogo.php
<?php
class Jogo{
    //Atributos
    private $id;
    private $nome;
    private $genero;
    private $ano;
    private $tenho;
    private $favorito;

    //Construtor
    function __construct($nome, $genero, $ano, $tenho = false, $favorito = false, $id = 0){
        $this -> id = $id;
        $this -> nome = $nome;
        $this -> genero = $genero;
        $this -> ano = $ano;
        $this -> tenho = $tenho;
        $this -> favorito = $favorito;
    }

    //Setters
    function setId($id){
        $this -> id = $id;
    }
    function setNome($nome){
        $this -> nome = $nome;
    }
    function setGenero($genero){
        $this -> genero = $genero;
    }
    function setAno($ano){
        $this -> ano = $ano;
    }
    function setTenho($tenho){
        $this -> tenho = $tenho;
    }
    function setFavorito($favorito){
        $this -> favorito = $favorito;
    }

    //Getters
    function getId(){
        return $this -> id;
    }
    function getNome(){
        return $this -> nome;
    }
    function getGenero(){
        return $this -> genero;
    }
    function getAno(){
        return $this -> ano;
    }
    function getTenho(){
        return $this -> tenho;
    }
    function getFavorito(){
        return $this -> favorito;
    }
    
}
?>